packages/hint-no-html-only-headers/src/hint.ts

```diff
@@ -37,12 +37,23 @@ export default class NoHtmlOnlyHeadersHint implements IHint {
             'content-security-policy',
             'feature-policy',
             'x-content-security-policy',
-            'x-frame-options',
             'x-ua-compatible',
             'x-webkit-csp',
             'x-xss-protection'
         ];
 
+        const exceptionHeaders: string[] = [
```

Comment:
Want to add a TODO to remove `exceptionHeaders` and `exceptionMediaTypes` from this hint entirely once we implement #25?